### DISCUSSION TITLE
大佬你好，请问为什么分辨率设置成640与1136

### DISCUSSION THREAD

**Discussion by Zhang-Yexun** [lisai9093/YYS#15] [Q&A] [answered]
原始分辨率一般是720*1280，但是确实设置成了640与1136，而且强调一定要这样设置。但是我想不通。所以能请问为什么要这样设置吗？我觉得分辨率的选择是一个很重要的问题，
因为这对于确保自动化脚本的触控操作能够准确发生在预期的屏幕位置上是非常关键的。

**Comment by lisai9093** [comment, marked as answer]
640x1136的分辨率是因为这是桌面版的默认分辨率。统一分辨率的好处就是一套截图可以用在桌面版和模拟器上，不需要再另外截图。当然也可以使用自动缩放去做兼容，但是之前的测试效果并不是很好。所以目前最好的解决方法就是强制统一分辨率，结束脚本后自动恢复。

**Reply by Sit0527** [reply]
那个我用咱这个项目试了一下然后分辨率变成640x1136，怎么改回去。。。？
mumu最新的。

**Reply by Sit0527** [reply]
现在重启电脑也还是卡在640x1136。。。

**Reply by Sit0527** [reply]
我可能是没断开adb这个操作导致的。

**Reply by lisai9093** [reply]
是的 断开会回到原分辨率

**Comment by Zhang-Yexun** [comment]
明白了，谢谢
